app/_components/stations/Form.tsx

```diff
@@ -0,0 +1,347 @@
+import powers from '@/app/_lib/power-list';
+import governments from '@/app/_lib/government-list';
+import economies from '@/app/_lib/economy-list';
+import allegiances from '@/app/_lib/allegiance-list';
+import factionStates from '@/app/_lib/faction-state-list';
+
+import {
+  Button,
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  Input,
+  FormErrorMessage,
+  Checkbox,
+} from '@chakra-ui/react';
+
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm, SubmitHandler } from 'react-hook-form';
+import useColorMode from '@/app/_hooks/useColorMode';
+import selectColor from '@/app/_hooks/fontColorSelector';
+import PowersField from '@/app/_components/inputs/Powers';
+import AllegiancesField from '@/app/_components/inputs/Allegiances';
+import GovernmentsField from '@/app/_components/inputs/Governments';
+import PowerEffectsField from '@/app/_components/inputs/PowerEffects';
+import FactionStatesField from '@/app/_components/inputs/FactionStates';
+import EconomiesField from '@/app/_components/inputs/Economies';
+import LandingPad from '@/app/_components/inputs/LandingPads';
+import StationTypes from '@/app/_components/inputs/StationTypes';
+import ShipsField from '@/app/_components/inputs/Ships';
+import ModulesField from '@/app/_components/inputs/Modules';
+import CommoditiesField from '@/app/_components/inputs/Commodities';
+import FacilitiesField from '@/app/_components/inputs/Facilities';
+import CheckboxGroup from '../form/CheckboxGroup';
```

Comment:
Can we add an `index.ts` to export all these modules from the inputs component folder so we can import then like `import { ShipsField, ModulesField } from '@/app/_components/inputs`;

Reply:
@nitishdhar I don't know if I'm missing something here, but Ive added them all to an index.ts and my vscode isn't picking it up for autocomplete. So as I start going through the forms and re-importing all the components, it's insisting the only - for example - AllegiancesField component is in `../inputs/Allegiances`

Is there something else I need to do here to get this to work? Because manually typing in imports feels like an awful trade off

![image](https://github.com/ed-pilots-network/frontend/assets/319883/dea320f8-5865-42d7-9cd5-25e6ac3fb861)

Reply:
I puzzled out a workaround but it's caused a bunch of new issues.  I can get autocomplete working by exporting as a named component, except then ES lint complains you should favour default components - so you end up having to do both:

```
export const AllegiancesField = ({ register, placeholder }: Props) => (
  <Select placeholder={placeholder} register={register}>
    {allegiances.map((item) => (
      <option key={item} value={item}>
        {item}
      </option>
    ))}
  </Select>
);

export default AllegiancesField;
```

Strange code aside, this then breaks the mocks in the tests and I can't puzzle out a way of fixing it. Obviously this won't work anymore:

```
jest.mock('../inputs/Powers', () => ({
  __esModule: true,
  default: jest
    .fn()
    .mockReturnValue(
      <input value="some value" onChange={(e) => e.target.value} />,
    ),
}));
```

But I can't mock `../inputs`, so a quick Google tells me the way around this is to use `jest.mocked` (https://jestjs.io/docs/mock-function-api/#jestmockedsource-options) but this doesn't work in our case, it reverts to throwing the same errors I mentioned in this review: https://github.com/ed-pilots-network/frontend/pull/77#discussion_r1289430271

So I'm not sure how to fix this. Ive spent a couple of hours investigating and I think any further time would be better spent trying to puzzle out why the react-select components are causing test errors

Reply:
I have pushed a commit that implements the barrel export pattern. You had to use `default as ComponentName` while retaining the default export.